{
  "uid" : "58725bc55e65351d",
  "name" : "TestCheckInit.test_with_negative_number",
  "title" : "Test with negative number",
  "time" : {
    "start" : 1525901683721,
    "stop" : 1525901683722,
    "duration" : 1
  },
  "summary" : {
    "attachments" : 0,
    "steps" : 0
  },
  "failure" : null,
  "description" : null,
  "severity" : "NORMAL",
  "status" : "PASSED",
  "testId" : null,
  "suite" : {
    "uid" : "6d45831bf6a3d4ff",
    "name" : "test_first",
    "title" : "Test first"
  },
  "steps" : [ ],
  "attachments" : [ ],
  "issues" : [ ],
  "labels" : [ {
    "name" : "story",
    "value" : "Checking the output with negative number"
  }, {
    "name" : "feature",
    "value" : "Incorrect input"
  }, {
    "name" : "suite-name",
    "value" : "test_first"
  }, {
    "name" : "thread",
    "value" : "7087-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "suite-title",
    "value" : null
  }, {
    "name" : "host",
    "value" : "familypc"
  }, {
    "name" : "severity",
    "value" : "normal"
  }, {
    "name" : "language",
    "value" : "cpython2"
  } ],
  "parameters" : [ ]
}